Dockerfile

```diff
@@ -1,23 +1,29 @@
-FROM golang:1.9.0-alpine3.6 AS build
+# GitHub:       https://github.com/gohugoio/
+# DockerHub:    https://hub.docker.com/r/gohugoio/
+# Quay:         https://quay.io/user/gohugoio
```

Comment:
I see some accounts above that I don't recognize. I think it is better to just leave the official 2.

Reply:
hi @bep,

like mentioned a while ago [here](https://github.com/gohugoio/hugo/pull/4155#issuecomment-350538197) i created those accounts to be consistent with the naming. just let me know when you consider to use them and i will hand them over to you ...

cheers maik

Reply:
@bep Updated to only official